js/package.json

```diff
@@ -56,7 +56,9 @@
     "webpack-cli": "^3.0.8"
   },
   "dependencies": {
-    "@jupyter-widgets/base": "^2.0.0",
+    "@jupyter-widgets/base": "^2 || ^3",
+    "@phosphor/messaging": "^1",
+    "@phosphor/widgets": "^1",
```

Comment:
Isn't it needed to use `lumino` now? I guess it will still compile and work just fine though.